packages/sit-onyx/src/styles/fonts.scss

```diff
@@ -17,3 +17,17 @@
     font-family: var(--onyx-font-family-mono);
   }
 }
+
+.onyx-truncation {
+  &-ellipsis {
+    white-space: nowrap;
+    overflow: hidden;
+    text-overflow: ellipsis;
+  }
+
+  &-multiline {
+    // needed to wrap the text to a newline even when its one long word,
+    // otherwise the max width would be overflown
+    overflow-wrap: anywhere;
```

Comment:
should we also enable hyphens then?
https://developer.mozilla.org/en-US/docs/Web/CSS/hyphens?retiredLocale=de

Reply:
I like the idea, lets quickly talk about it in the daily